README.md

```diff
@@ -675,3 +714,24 @@ Your changes will only take effect in new terminal windows.
 [running_gif]: doc/images/opensim_running.gif
 [simple_example_gif]: doc/images/opensim_double_pendulum_muscle.gif
 [java]: http://www.oracle.com/technetwork/java/javasebusiness/downloads/java-archive-downloads-javase6-419409.html
+
+#### For the impatient (Ubuntu)
```

Comment:
Could you list which version of Ubuntu these instructions are for? On more recent versions of Ubuntu, users probably don't need to add a ppa for cmake.